.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] no changes have been made in relation to the styling of the pop-up menu as requested by the previous CR, in case you don't understand the requirements please leave a message in the pull request or ask for a 1 on 1 session with a senior CR.
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/225368488-0587832f-3810-43db-a525-eb91802ed405.png)
figma design:
![afbeelding](https://user-images.githubusercontent.com/39240820/225368698-a41e1e01-a056-4155-ba3d-d81609ea58fd.png)